shortcircuit-api/src/main/java/com/powsybl/shortcircuit/json/FaultResultDeserializer.java

```diff
@@ -91,6 +94,12 @@ public FaultResult deserialize(JsonParser parser, DeserializationContext deseria
                     shortCircuitBusResults = parser.readValueAs(new TypeReference<List<ShortCircuitBusResults>>() { });
                     break;
 
+                case "status":
+                    JsonUtil.assertGreaterOrEqualThanReferenceVersion(CONTEXT_NAME, "Tag: " + parser.getCurrentName(), ShortCircuitAnalysisResult.VERSION, "1.1");
```

Comment:
Since you have to check the current version, it can not be a `StdDeserializer`, just an object deserializer called in `ShortCircuitAnalysisResultDeserializer` with the current version passed as parameter